### ISSUE TITLE
TypeError: Headers are immutable

### ISSUE THREAD

**Issue by mxnkarou** [socketio/socket.io-deno#2]
Trying to setup Websockets with Deno & IO.
Upon establishing connection I'm getting this pesky error - any idea how to fix this?

```
TypeError: Headers are immutable.
    at Headers.set (deno:ext/fetch/20_headers.js:384:15)
    at https://deno.land/x/socket_io@0.1.0/packages/engine.io/lib/transports/websocket.ts:60:24
    at Headers.forEach (deno:ext/webidl/00_webidl.js:1001:13)
    at WS.onRequest (https://deno.land/x/socket_io@0.1.0/packages/engine.io/lib/transports/websocket.ts:59:21)
    at Server.handleRequest (https://deno.land/x/socket_io@0.1.0/packages/engine.io/lib/server.ts:217:35)
    at async Server.#respond (https://deno.land/std@0.150.0/http/server.ts:298:18)
 ```
 Server:
```
const io = new Server({ cors: { origin: 'http://localhost:3000' } });

io.on('connection', socket => {
  console.log(`Socket ${socket.id} connected`);

  socket.on('disconnect', reason => {
    console.log(`Socket ${socket.id} disconnected due to ${reason}`);
  });
});

await serve(io.handler(), {
  port: 8080,
});
```

Client:
```
const socket = io('ws://localhost:8080');

  socket.on('connection', () => {
    console.log('Connection established.');
  });
```

**Comment by darrachequesne** [comment]
Hi, thanks a lot for the feedback!

This should already be fixed in version [0.1.0](https://github.com/socketio/socket.io-deno/releases/tag/0.1.1) (by https://github.com/socketio/socket.io-deno/commit/dfe3122865e768ae75e1d4d8c92a47961d708ee9), could you please check?

**Comment by mxnkarou** [comment]
True!
Upgrading to 0.1.1 did the trick!
